packages/babel-helper-member-expression-to-functions/src/index.ts

```diff
@@ -193,10 +198,15 @@ const handle = {
       const parentIsOptionalCall = parentPath.isOptionalCallExpression({
         callee: node,
       });
+      // here we use a function to wrap `parentIsOptionalCall` to get type
+      // for parent, do not use it anywhere else
```

Comment:
Could you link https://github.com/microsoft/TypeScript/issues/10421 in this comment? :pray: